src/Uno.UI/UI/Xaml/Controls/ToolTip/ToolTipService.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Input;
+
+namespace Windows.UI.Xaml.Controls
+{
+	partial class ToolTipService
+	{
+		public static DependencyProperty ToolTipProperty { get; } =
+			DependencyProperty.RegisterAttached(
+				"ToolTip",
+				typeof(object),
+				typeof(ToolTipService),
+				new FrameworkPropertyMetadata(default, OnToolTipChanged));
+
+		public static object GetToolTip(DependencyObject element)
+		{
+			return element.GetValue(ToolTipProperty);
+		}
+
+		public static void SetToolTip( global::Windows.UI.Xaml.DependencyObject element, object value)
+		{
+			element.SetValue(ToolTipProperty, value);
+		}
+
+		private static void OnToolTipChanged(DependencyObject dependencyobject, DependencyPropertyChangedEventArgs args)
+		{
+			if (!(dependencyobject is FrameworkElement element))
+			{
+				return;
+			}
+
+			if (args.NewValue is ToolTip toolTip)
```

Comment:
What if it's set to a string? Eg in the Calculator https://github.com/unoplatform/calculator/blob/uno/src/Calculator.Shared/Resources/en-US/Resources.resw#L828

Reply:
You're right, an implicit converter is missing here.